lib/events/simplestateevents.h

```diff
@@ -71,4 +71,21 @@ class RoomAliasesEvent
     QStringList aliases() const { return content().value; }
 };
 REGISTER_EVENT_TYPE(RoomAliasesEvent)
+
+class RoomPinnedEvent
+    : public StateEvent<EventContent::SimpleContent<QStringList>>
+{
+public:
+    DEFINE_EVENT_TYPEID("m.room.pinned_messages", RoomPinnedEvent)
+
+    explicit RoomPinnedEvent(const QJsonObject& json)
+        : StateEvent(typeId(), json, QStringLiteral("pinned"))
+    { }
+    explicit RoomPinnedEvent(const QStringList& roomEvents)
+        : StateEvent(typeId(), matrixTypeId(), {},
+                     QStringLiteral("pinned"), roomEvents)
+    { }
+    QStringList pinnedEvents() const { return content().value; }
+};
+REGISTER_EVENT_TYPE(RoomPinnedEvent)
```

Comment:
Do you skip on using `DEFINE_SIMPLE_STATE_EVENT` macro just to name the accessor differently? If not, can you point me to what I'm missing.